modules/oxide/src/less/theme/components/slider/slider.less

```diff
@@ -53,3 +53,36 @@
     width: @slider-handle-width;
   }
 }
+
+.tox:not([dir=rtl]) {
```

Comment:
Can this be done direction-agnostic and eliminate duplication by utilising `margin-inline-start` instead? 

Using `gap` property for flex element might also be an option (as well as potentially removing the need for the `:not(:first-of-type)` kind of selectors), but browser support is a bit worse because of quite late adoption by Safari.

Reply:
This is also just coming from the existing image tools styling. It is a good opportunity to improve the styles, though.

https://github.com/tinymce/tinymce/blob/c033cc53ed381bf3f4b3701d4c846061be0d7bac/modules/oxide/src/less/theme/components/image-tools/image-tools.less#L102-L133

Reply:
Looks like that works as `margin-inline-start`, so I'll go with it

Reply:
Possibly worth noting that we couldn't use `margin-inline-start` previously as we had to support IE 11. That's no longer the case now so we should be safe to use it now.

Reply:
ah yep I saw that when I was checking MDN but forgot to note it here.